src/pkg/docs/model/attributes.go

```diff
@@ -19,14 +22,24 @@ const (
 	AttributeTimestamp       Attribute = 12
 )
 
-type Attributes []ehrIndexer.AttributesAttribute
+type AttributesEhr []ehrIndexer.AttributesAttribute
 
-func (a Attributes) GetByCode(code Attribute) []byte {
+func (a AttributesEhr) GetByCode(code Attribute) []byte {
 	for _, attr := range a {
 		if attr.Code == code {
 			return attr.Value
 		}
 	}
+	return nil
+}
 
+type AttributesUsers []users.AttributesAttribute
+
+func (a AttributesUsers) GetByCode(code Attribute) []byte {
```

Comment:
Maybe Im  mistaken, but i think it will be better to do it like that: 
```go
type Attributes []ehrIndexer.AttributesAttribute

type Attributer interface {
	GetByCode(code Attribute) []byte
}

type UserAttributes struct {
	Attributes
}

type AttributesEhr struct {
	Attributes
}

func (a Attributes) GetByCode(code Attribute) []byte {
	for _, attr := range a {
		if attr.Code == code {
			return attr.Value
		}
	}

	return nil
}

```

Reply:
Unfortunately, it doesn't work that way. We'll leave it for better times.